horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/TestServiceImpl.java

```diff
@@ -103,6 +104,15 @@ SELECT DISTINCT COALESCE(jsonb_object_agg(label.name, lv.value) FILTER (WHERE la
          ) select * from combined FILTER_PLACEHOLDER ORDER_PLACEHOLDER limit :limit offset :offset
          """;
 
+   protected static final String LABEL_VALUES_SUMMARY_QUERY = """
+         SELECT DISTINCT COALESCE(jsonb_object_agg(label.name, lv.value), '{}'::::jsonb) AS values
```

Comment:
As you highlighted here https://github.com/Hyperfoil/Horreum/issues/1766 I think you can use `::` and it should work as well
```suggestion
         SELECT DISTINCT COALESCE(jsonb_object_agg(label.name, lv.value), '{}'::jsonb) AS values
```